src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
Also, @whipermr5, update: The following CSS seems to work as well and is a LOT less verbose than what I posted earlier:

```CSS
.alert-success:before {
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-top: .15em;
    padding-right: .5em;
    content: '\e084';
    float: left;
}

.alert-success:not(.modal-header):before {
    padding-top: 0;
}
```